site/static/examples/multikueue/tekton-pipelinerun-example.yaml

```
@@ -0,0 +1,114 @@
+kind: PipelineRun
+metadata:
+  name: example-pipeline
+  namespace: default
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+spec:
+  managedBy: "kueue.x-k8s.io/multikueue"  # This tells Kueue to manage this PipelineRun
+  pipelineRef:
+    name: example-pipeline
+  params:
+  - name: message
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)